website/server/models/group.js

```diff
@@ -1101,7 +1103,7 @@ schema.statics.processQuestProgress = async function processQuestProgress (user,
 
   let questType = quest.boss ? 'Boss' : 'Collection';
 
-  await group[`_process${questType}Quest`]({
+  await group[`_process${questType}Quest`]({ // processBossQuest, processCollectionQuest
```

Comment:
I added this comment because without it there's no way to find where those two functions are called unless you remember that the function names are created on the fly.

Reply:
sounds good but I think it should have an underscore in front of the methods so if you're searching for the full method name like `_ processBossQuest` thill will get matched! Then feel free to merge

Reply:
Ha, of course, you'd think I would have thought of that. :) Thanks! I've changed it and I'll merge when the tests pass.